pyproject.toml

```diff
@@ -29,18 +29,65 @@ dependencies = [
     "tiktoken>=0.4.0",
 ]
 description = "LLM Chain for answering questions from docs"
-dynamic = ["optional-dependencies", "version"]
+dynamic = ["version"]
 keywords = ["question answering"]
 license = {file = "LICENSE"}
 maintainers = [
     {email = "jamesbraza@gmail.com", name = "James Braza"},
+    {email = "michael.skarlinski@gmail.com", name = "Michael Skarlinski"},
     {email = "white.d.andrew@gmail.com", name = "Andrew White"},
 ]
 name = "paper-qa"
 readme = "README.md"
 requires-python = ">=3.8"
 urls = {repository = "https://github.com/whitead/paper-qa"}
 
+[project.optional-dependencies]
+agents = [
+    "anyio",
+    "langchain-community",
+    "langchain-openai",
+    "pymupdf",
+    "requests",
+    "tantivy",
+    "tenacity",
+    "typer",
+    "typing_extensions",
+]
+dev = [
+    "build",
+    "mypy",
+    "pre-commit",
+    "pytest",
+    "pytest-asyncio",
+    "pytest-subtests",
+    "pytest-sugar",
+    "pytest-timer",
+    "pytest-vcr",
+    "python-dotenv",
+    "pyzotero",
```

Comment:
Should `pyzotero ` live in `dev`? Maybe it should live in `llms`

Reply:
It's not an llm thing -- it's really an extension of its own. I'm leaving in dev for backwards compatibility.